ignite/contrib/handlers/clearml_logger.py

```diff
@@ -282,6 +298,7 @@ def global_step_transform(*args, **kwargs):
             Default is None, global_step based on attached engine. If provided,
             uses function output as global_step. To setup global step from another engine, please use
             :meth:`~ignite.contrib.handlers.clearml_logger.global_step_from_engine`.
+        state_attributes: list of attributes of the ``trainer.state`` to plot.
```

Comment:
Please add `versionchanged` after the Note as explained here: https://github.com/pytorch/ignite/pull/2137#issuecomment-895112574

Reply:
I have added ``versionchanged`` here, if it is fine I'll update the other two also.

Reply:
No, you added it in the wrong place. It should be in the end of the docstring and make sure to respect indents like here: https://github.com/pytorch/ignite/blob/c6ff5b8533e4c9b0dbd0c2b89606edf9a1ea8647/ignite/handlers/stores.py#L38-L39